### ISSUE TITLE
🐞 Fix bad layout issues on short single page articles

### ISSUE THREAD

**Issue by breck7** [publicdomaincompany/scroll#32]
[<img alt="Issuehunt badges" src="https://img.shields.io/badge/IssueHunt-%2450%20Funded-%2300A156.svg" />](https://issuehunt.io/r/publicdomaincompany/scroll/issues/32)

<img width="1356" alt="Screen Shot 2021-05-19 at 6 30 22 AM" src="https://user-images.githubusercontent.com/74692/118849954-f5d01b80-b86b-11eb-90e1-c127f5b3de63.png">

---

<details>
<summary>
<b>IssueHunt Summary</b>
</summary>

### Backers (Total: $50.00)

- [<img src='https://avatars.githubusercontent.com/u/74692?v=4' alt='breck7' width=24 height=24> breck7](https://issuehunt.io/u/breck7) ($50.00)

### Tips

- Checkout the [Issuehunt explorer](https://issuehunt.io/r/publicdomaincompany/scroll/) to discover more funded issues.
- Need some help from other developers? [Add your repositories](https://issuehunt.io/r/new) on IssueHunt to raise funds.
</details>

**Comment by AlaricBaraou** [comment]
in scroll.hakon 
```
.scrollArticleDate
 font-style italic
 font-size 80%
 display inline-block
 text-align left
```

For the CSS issue with column I couldn't figure out a way to do what you asked for.

Maybe redoing it with flexbox using flex-wrap and flex-direction column could do the trick but I'm not 100% and that would mean too much change. 

Nevermind the bounty, it was a fun brain teaser hope this still help.

Good luck !

**Comment by breck7** [comment]
The date ended up being a no code fix. I just moved the `date ` line below the `image` line and that solves it.

For the minimum column height, there might not be a good way to do it yet in CSS.

https://github.com/w3c/csswg-drafts/issues/3224

Perhaps at build time can make an estimate of the size of the article and if it doesn't pass some threshold force it to single column.

**Comment by taherdhanera** [comment]
Submitted a focused fix in PR #163: https://github.com/breck7/scroll/pull/163
